app/webfront/src/components/Profile/index.jsx

```diff
@@ -83,13 +84,13 @@ export default function Profile({ isMobile }) {
 
     return (
         <div>
-            <UserMenu onLogoutOpen={() => setShowLogoutModal(true)}/>
+            <UserMenu onLogoutOpen={() => setShowLogoutModal(true)} />
             {showLogoutModal && (
                 <Modal onClose={() => setShowLogoutModal(false)}>
-                    <h2>ARE YOU LOGGING OUT?</h2>
-                    <p>You can log back in at any time.</p>
-                    <button onClick={handleLogout}>Logout</button>
-                    <button onClick={() => setShowLogoutModal(false)}>Cancel</button>
+                    <h2 className={styles.modalTitle + ' ' + styles.center + ' ' + styles.modalTitleLogout}>ARE YOU LOGGING OUT?</h2>
```

Comment:
Щоб не було такої ковбаси краще користуватись clsx 

```suggestion
                    <h2 className={clsx(styles.modalTitle, styles.center, styles.modalTitleLogout)}>ARE YOU LOGGING OUT?</h2>
```

ну і для якоїсь послідовності не styles а css

Reply:
Звісно гарніше, не користувався. Зроблю.